testing/vcs/CMakeLists.txt

```diff
@@ -287,7 +287,7 @@ cdat_add_test(vcs_test_taylor_2_quads
       "${PYTHON_EXECUTABLE}"
       ${cdat_SOURCE_DIR}/testing/vcs/test_vcs_show.py
       )
-    FOREACH(gm boxfill isofill isoline vector meshfill )
+    FOREACH(gm boxfill isofill isoline meshfill )
```

Comment:
it's only the proejction vectors plot that are disabled for test, regular proj tests for vectors are still in